graphql/project_resolver.go

```diff
@@ -56,3 +56,13 @@ func (r *projectResolver) Patches(ctx context.Context, obj *restModel.APIProject
 func (r *Resolver) Project() ProjectResolver { return &projectResolver{r} }
 
 type projectResolver struct{ *Resolver }
+
+// !!! WARNING !!!
+// The code below was going to be deleted when updating resolvers. It has been copied here so you have
+// one last chance to move it out of harms way if you want. There are two reasons this happens:
+//   - When renaming or deleting a resolver the old code will be put in here. You can safely delete
+//     it when you're done.
+//   - You have helper methods in this file. Move them out to keep these resolver files clean.
```

Comment:
Should we keep this warning long-term, or can we delete it? If we plan to keep it temporarily, is there a ticket to delete it?

Reply:
Oops sorry I think i didn't fully fix graphql things. Let me take another pass.